src/sqlfluff/dialects/dialect_hive.py

```diff
@@ -556,3 +559,30 @@ class IntervalExpressionSegment(BaseSegment):
             ),
         ),
     )
+
+
+@hive_dialect.segment()
+class MsckRepairTableStatementSegment(BaseSegment):
+    """A repair table statement.
+
+    `MSCK REPAIR` reference here within Language Manual DDL page:
+    https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL
```

Comment:
```suggestion
    https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-RecoverPartitions(MSCKREPAIRTABLE)
```
More specific link to the relevant docs

Reply:
Hey @jpy-git! `MSCK` just denotes "metastore consistency check". I'm just trying to capture the `REPAIR TABLE` syntax in this segment but can create a separate `MSCK TABLE` for Hive just to capture delta in metadata between the file system and metastore. `MSCK TABLE` is essentially a dry run for `MSCK REPAIR TABLE` in the scenario there's a significant difference then you can use that to adjust cluster-size, etc. for the actual operation.

Let me know your thoughts on this approach :)

Reply:
@R7L208 actually let's take your approach and keep the `MsckRepairTableStatementSegment`s as is 👍 

Could you create a separate `MsckTableStatementSegment` statement for the no `REPAIR` case?
And add a note to the hive `MsckTableStatementSegment` docstring noting that there's a separate statement for that case 😄